### PR TITLE
Move hgupdate-sync label functionality to SyncLabelBot

### PR DESCRIPTION
**Pull request by rwestberg**
Move the responsibility of updating the hgupdate-sync label to the SyncLabelBot.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Reviewers
 * [Erik Helin](https://openjdk.java.net/census#ehelin) (@edvbld - **Reviewer**) ⚠️ Review applies to c60e5f3ab24608b29eaa2af564e68b2709c61fb6
 * [Magnus Ihse Bursie](https://openjdk.java.net/census#ihse) (@magicus - no project role)

### Download
`$ git fetch https://git.openjdk.java.net/skara pull/1021/head:pull/1021`
`$ git checkout pull/1021`

### CODE REVIEW

File: bots/synclabel/src/main/java/org/openjdk/skara/bots/synclabel/SyncLabelBotCheckWorkItem.java [hunk 1]
```diff
@@ -32,12 +32,12 @@
 
 public class SyncLabelBotCheckWorkItem implements WorkItem {
```
Comment (magicus) [inline bots/synclabel/src/main/java/org/openjdk/skara/bots/synclabel/SyncLabelBotCheckWorkItem.java:33]: Now that it is not only checking, but also updating the labels, maybe rename the class?

**Review by edvbld — APPROVED** [review]
Looks good modulo @magicus comment :+1:

File: bots/synclabel/src/main/java/org/openjdk/skara/bots/synclabel/SyncLabelBotCheckWorkItem.java (hunk 1 above)
Comment (rwestberg) [reply bots/synclabel/src/main/java/org/openjdk/skara/bots/synclabel/SyncLabelBotCheckWorkItem.java:33]: Makes sense, will change it.

**Review by magicus — APPROVED** [review]
Looks good!

**Pull request closed without merging**